### DISCUSSION TITLE
Creating a custom External Template for the @wordpress/create-block package

### DISCUSSION THREAD

**Discussion by ryanwelcher** [WordPress/developer-blog-content#161] [Topic Ideas]
[External Templates](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/packages-create-block-external-template/) are a very powerful tool to allow complete customization of block structure that is not widely known.

The article will focus on the technical steps of creating a template but from the use-case of an extender wanting to add more blocks to an existing plugin with pre-set values.

At the end of the article, I'd like to add a call for templates to allow the extender community to share their templates.

**Comment by bph** [comment]
The topic was approved in today's Editorial Group Meeting.
The discussion is locked, please continue the conversation on the issue

- #166

**Comment by ryanwelcher** [comment]
Post is live - https://developer.wordpress.org/news/2024/04/16/creating-an-external-project-template-for-create-block/
